translate/storage/test_ts2.py

```diff
@@ -60,6 +60,12 @@
 &apos;&quot;&apos;)</source>
         <translation type="unfinished"></translation>
     </message>
+    <message>
+        <source>Line 1
+
+Line 3</source>
+        <translation type="unfinished"></translation>
+    </message>
```

Comment:
You're adding test for ts2 storage, but patching ts storage, that looks like a mismatch.